dev-python/pipenv/pipenv-2021.5.29-r1.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DISTUTILS_USE_SETUPTOOLS=rdepend
-PYTHON_COMPAT=( python3_{7,8,9} )
+PYTHON_COMPAT=( python3_{7,8,9,10} )
```

Comment:
If going to touch this, remove 7 at same time. It's dead code that does nothing (not recognized by eclasses).

If adding 10, I assume you've ran the test suite with it?

Reply:
What do you mean by that? I don't understand.
Is Python3_7 dead or Python3_10?

Reply:
I ran the tests with Python3_10, they work.

Reply:
3_7 is dead as far as eclasses are concerned, so can cleanup

3_10 I'm just asking if you've tested it or just adding it like that

Reply:
Removed Python3.7